src/main/resources/lang/default.lang

```diff
@@ -468,867 +468,869 @@ directions:
 	command minecart:
 		name: command minecart¦s
-		pattern: command [mine]cart(|1¦s)|[mine]cart(|1¦s) with command block[s]
+		pattern: command [mine]cart[plural:s]|[mine]cart[plural:s] with command block[s]
 	mooshroom:
 		name: mooshroom¦s
-		pattern: <age> (mooshroom[ cow]|mushroom cow)(|1¦s)
+		pattern: <age> (mooshroom[ cow]|mushroom cow)[plural:s]
 	red mooshroom:
 		name: red mooshroom¦s
-		pattern: <age> red (mooshroom[ cow]|mushroom cow)(|1¦s)|(4¦)mooshroom (kid(|1¦s)|child(|1¦ren))
+		pattern: <age> red (mooshroom[ cow]|mushroom cow)[plural:s]|(baby:)mooshroom (kid[plural:s]|child[plural:ren])
 	brown mooshroom:
 		name: brown mooshroom¦s
-		pattern: <age> brown (mooshroom[ cow]|mushroom cow)(|1¦s)|(4¦)mooshroom (kid(|1¦s)|child(|1¦ren))
+		pattern: <age> brown (mooshroom[ cow]|mushroom cow)[plural:s]|(baby:)mooshroom (kid[plural:s]|child[plural:ren])
 	ocelot:
 		name: ocelot¦s @an
-		pattern: <age> ocelot(|1¦s)
+		pattern: <age> ocelot[plural:s]
 	wild ocelot:
 		name: wild ocelot¦s
-		pattern: (wild|untamed) <age> ocelot(|1¦s)
+		pattern: (wild|untamed) <age> ocelot[plural:s]
 	cat:
 		name: cat¦s
-		pattern: <age> [%-cattype%] cat(|1¦s)|tamed <age> ocelot(|1¦s)|(4¦)[%-cattype%] kitten(|1¦s)
+		pattern: <age> [%-cattype%] cat[plural:s]|tamed <age> ocelot[plural:s]|(baby:)[%-cattype%] kitten[plural:s]
 	painting:
 		name: painting¦s
-		pattern: painting(|1¦s)
+		pattern: painting[plural:s]
 	pig:
 		name: pig¦s
-		pattern: <age> [%-pigvariant%] pig(|1¦s)|(4¦)piglet(|1¦s)
+		pattern: <age> [%-pigvariant%] pig[plural:s]|(baby:)piglet[plural:s]
 	saddled pig:
 		name: saddled pig¦s
-		pattern: saddled [%-pigvariant%] pig(|1¦s)
+		pattern: saddled [%-pigvariant%] pig[plural:s]
 	unsaddled pig:
 		name: unsaddled pig¦s @an
-		pattern: unsaddled [%-pigvariant%] pig(|1¦s)
+		pattern: unsaddled [%-pigvariant%] pig[plural:s]
 	player:
 		name: player¦s
-		pattern: player(|1¦s)
+		pattern: player[plural:s]
 	zombie pigman:
 		name: zombie pig¦man¦men|zombified piglin
-		pattern: <age> (zombie pigm(an|1¦en)|zombified piglin(|1¦s))|(4¦)(zombie pigletboy(|1¦s)|zombified piglin (kid(|1¦s)|child(|1¦ren)))
+		pattern: <age> (zombie pigm(an|plural:en)|zombified piglin[plural:s])|(baby:)(zombie pigletboy[plural:s]|zombified piglin (kid[plural:s]|child[plural:ren]))
 	sheep:
 		name: sheep
 		# while sheep is ambiguous, sheeps is plural (even though it's wrong)
-		pattern: [%-colors%] <age> sheep(2¦|1¦s)|(4¦)[%-colors%] lamb(|1¦s)
+		pattern: [%-colors%] <age> sheep(unknown_plural:|plural:s)|(baby:)[%-colors%] lamb[plural:s]
 	unsheared sheep:
 		name: unsheared sheep @an
-		pattern: unsheared [%-colors%] sheep(2¦|1¦s)
+		pattern: unsheared [%-colors%] sheep(unknown_plural:|plural:s)
 	sheared sheep:
 		name: sheared sheep
-		pattern: sheared [%-colors%] sheep(2¦|1¦s)
+		pattern: sheared [%-colors%] sheep(unknown_plural:|plural:s)
 	silverfish:
 		name: silverfish¦es
-		pattern: silverfish(|1¦es)
+		pattern: silverfish[plural:es]
```

Comment:
I might consider an unknown plural here for just `silverfish`, `five silverfish` reads normally